### ISSUE TITLE
Screencopy always renders cursor on headless backend

### ISSUE THREAD

**Issue by any1** [swaywm/wlroots#2320]
The protocol specifies that overlaying of the cursor can be disabled. This works on the DRM backend, but it is not implemented on headless.

See https://github.com/any1/wayvnc/issues/47

**Comment by emersion** [comment]
Yeah, currently wlroots only supports stripping the cursor if hardware cursors are supported by the backend.
